model/cicd/spans.yaml

```diff
@@ -0,0 +1,26 @@
+groups:
+  - id: span.cicd.pipeline.run.server
+    type: span
+    span_kind: server
+    brief: 'Semantic Convention for pipeline runs'
```

Comment:
```suggestion
    brief: This span describes CICD pipeline run
```

Reply:
Done in 9298f48736e4b56fa3678c241e304c4412fa428a.